src/styles/variables.scss

```diff
@@ -1,15 +1,21 @@
 :root {
-  --color-white: #f1f2f9;
-  --color-accent: #905bff;
-  --color-green: #27ae60;
-  --color-red: #eb5757;
+  --primeTextColor: #f1f2f9;
+  --secondaryTextColor: #75767f;
 
-  --color-black: #0f1121;
-  --color-surface-one: #161827;
-  --color-surface-two: #323542;
-  --color-elements: #3b3e4a;
-  --color-icons: #4a4d58;
-  --color-secondary: #75767f;
+  --primeAccentColor: #905bff;
+  --secondaryAccentColor: #a378ff;
```

Comment:
this is wrong color
it should be green, but purple  (its wrong name in the design)
```suggestion
  --secondaryAccentColor: #27ae60;
```

![image](https://github.com/fe-jul23-team6/product_catalog/assets/108063043/91848aa4-85b4-4ba3-ad8e-b9873f31c33c)